Dockerfile

```diff
@@ -19,8 +19,7 @@ WORKDIR /home/docker/hpe-oneview-hubot
 
 RUN cp -rf /home/docker/hubot-core-org/node_modules/ /home/docker/hpe-oneview-hubot/
 
-COPY gulpfile.babel.js /home/docker/hpe-oneview-hubot/
+COPY gulpfile.js /home/docker/hpe-oneview-hubot/
```

Comment:
Does it work if you remove this line? (COPY gulpfile.js)

Reply:
We need to copy the gulp file so that we can build the software, it should stay.